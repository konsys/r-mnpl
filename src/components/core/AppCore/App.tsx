import "fontsource-roboto";
import "./app.scss";
import "../../../theme/styles/vars.scss";

import { Route, BrowserRouter as Router, Switch } from "react-router-dom";

import { Board } from "../BoardCore/Board";
import { Game } from "../Game/Game";
import GameChatSocket from "socket/GameChatSocket";
import GameModal from "../Game/GameModal/GameModal";
import { Login } from "../Registration/Login/Login";
import { MuiThemeProvider } from "@material-ui/core";
import NotFound from "../../views/Errors/NotFound";
import React from "react";
import RoomsSocket from "socket/RoomsSocket";
import TopFivePage from "../Game/TopFivePage/TopFivePage";
import { theme } from "../../../theme";

const App = () => {
  return (
    <>
      {/* <React.StrictMode> */}
      <MuiThemeProvider theme={theme}>
        <GameModal />
        <GameChatSocket />
        <RoomsSocket />
        <Router>
          <Switch>
            <Route exact path="/login" component={Login} />
            <Route exact path="/board/:id" component={Board} />
            <Route exact path="/games" component={Game} />
            <Route path="/top-five" component={TopFivePage} />
            <Route path="/" component={NotFound} />
          </Switch>
        </Router>
      </MuiThemeProvider>
      {/* </React.StrictMode> */}
    </>
  );
};

export default App;
