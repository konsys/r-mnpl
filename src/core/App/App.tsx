import React from "react";
import "./App.css";
import { Router } from "@reach/router";
import { Game } from "../Game";

function App() {
  return (
    <>
      <Router>
        <Game path="/game" />
      </Router>
    </>
  );
}

export default App;
