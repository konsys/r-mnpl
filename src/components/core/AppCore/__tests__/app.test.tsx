import App from "../App";
import React from "react";
import { shallow } from "enzyme";

describe("App test", () => {
  it("should render", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
