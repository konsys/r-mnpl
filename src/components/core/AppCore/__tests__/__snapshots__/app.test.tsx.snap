// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should render 1`] = `ShallowWrapper {}`;
